packages/orchestration/src/cosmos-api.ts

```diff
@@ -101,25 +102,28 @@ export interface CosmosAssetInfo extends Record<string, unknown> {
 /**
  * Info for a Cosmos-based chain.
  */
-export type CosmosChainInfo = Readonly<{
+export interface CosmosChainInfo extends BaseChainInfo {
   /** can be used to lookup chainInfo (chainId) from an address value */
-  bech32Prefix?: string;
-  chainId: string;
-
-  connections?: Record<string, IBCConnectionInfo>; // chainId or wellKnownName
+  readonly bech32Prefix?: string;
```

Comment:
the `readonly` here makes it harder to read and unnecessarily precludes a mutation pattern.

Please remove these and instead use the `Readonly<T>` utility when relevant. E.g. below,
```js
export type ChainInfo = Readonly<BaseChainInfo | CosmosChainInfo | EIP155ChainInfo>;
```